### PR TITLE
Added 418 and 508

### PR DESCRIPTION
**Pull request by panic175**
How can this project exist without teapots? @liebig

### CODE REVIEW

**Comment by liebig** [comment]
I'm looking forward for a soon merge!

**Comment by liebig** [comment]
@panic175  Please add tests and documentation for your code changes!

File: index.js [hunk 1]
```diff
@@ -16,8 +16,10 @@ module.exports = {
   405: '❗',
   409: '💥',
   410: '💨',
+  418: '☕',
```
Comment (bendrucker) [inline index.js:19]:
```suggestion
  418: '🍵',
```

**Review by bendrucker — CHANGES_REQUESTED** [review]
Made my day to see this 😄

**Pull request closed without merging**